domains/devendra.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devendrakanojiya",
+        "email": "s_36100@bkbirlacollegekalyan.com"
+    },
+    "record": {
+        "CNAME": "devendrakanojiya.github.io"
```

Comment:
page gives 404 page

Reply:
my github page is running on  https://devendrakanojiya.github.io/me

but as far I know we don't have to add "/me" in record  or we have to ?

Reply:
fixed it 👍